Security/PasswordResetToken.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Security;
+
+use SumoCoders\FrameworkMultiUserBundle\Exception\InvalidPasswordResetTokenException;
+use SumoCoders\FrameworkMultiUserBundle\User\UserInterface;
+
+class PasswordResetToken
+{
+    private function generatePasswordResetToken(){
```

Comment:
hebt ge hier geen pre commit op staan? Tzou toffer zijn moesten code styles in de initiele commits direct goed zijn, en er dus niet overal "code style" commits tussen zouden zitten.